Makefile

```diff
@@ -41,14 +41,17 @@ APP_VER		:= $(shell git describe 2> /dev/null || echo "unknown")
 REGISTRY_APP_NAME		:= a8registry
 CONTROLLER_APP_NAME		:= a8controller
 SIDECAR_APP_NAME		:= a8sidecar
+K8SRULES_APP_NAME		:= a8k8srulescontroller
 
```

Comment:
I would request that we don't run multiple independent and unrelated code in the same process, controlled via command line switches. Let's keep them as separate processes. I can see how it might complicate docs and docker image generation, but taking this argument to its (unreasonable) conclusion implies that we should have all functions (controller, registry, proxy, ...) in a single image and allow selection via invocation flags.